doc/source/math/resizing_ops.rst

```diff
@@ -0,0 +1,319 @@
+.. _resizing_ops:
+
+##################
+Resizing Operators
+##################
+
+
+Introduction
+============
+In ODL, resizing of a discretized function is understood as the operation of shrinking or enlarging its domain in such a way that the size of the partition cells do not change.
+This "constant cell size" restriction is intentional since it ensures that the underlying operation can be implemented as array resizing without resampling, thus keeping those two functionalities separate.
+
+
+Basic setting
+=============
+Let now :math:`\mathbb{R}^n` with :math:`n \in \mathbb{N}` be the space of one-dimensional real vectors encoding values of a function defined on an interval :math:`[a, b] \subset \mathbb{R}`.
+Since values are not manipulated, the generalization to complex-valued functions is straightforward.
+Further, since all operations completely separate with respect to different dimensions, the results can easily applied to the multi-dimensional case.
+
+
+Restriction operator
+====================
+We consider the space :math:`\mathbb{R}^m` for an :math:`m < n \in \mathbb{N}` and define the restriction operator
```

Comment:
This could perhaps be done while allowing `m = n`, this would allow some useful cases such as the trivial restriction.